app/controllers/admin/timeline_step_moves_controller.rb

```diff
@@ -27,6 +27,14 @@ def create
     render nothing: true
   end
 
+  # A change of date for a step that does not require a corresponding log
```

Comment:
@hooverlunch: Is this enough for a message? I spoke with linesarefuzzy about adding this method to this file specifically. There's a bit more going on here than just the saving of the log. The other methods refer to the Timeline StepMove model, but that's not needed here. We are simply changing a date in this method.